app/presenters/publishing_api/payload_builder/body_headings.rb

```diff
@@ -12,6 +12,8 @@ def initialize(item)
       end
 
       def call
+        return {} if body_headings.empty?
```

Comment:
How heavy is the call to Govspeak::Document.new in body_headings? Should we store the value of body_headings it in a temp variable here to avoid calling it twice?

Reply:
Yes good point! This would be an issue with very long documents - I've updated this now.